lib/pages/home/home_page.dart

```diff
@@ -21,32 +20,35 @@ class HomePage extends StatelessWidget {
               padding: const EdgeInsets.symmetric(horizontal: 24),
               child: Wrap(
                 children: [
-                  Row(
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    children: [
-                      // TODO: Bind real data (date & user profile) later in [Integrate].
-                      Column(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: [
-                          Text(
-                            DateFormat('EEEE, MMMM dd')
-                                .format(DateTime.now())
-                                .toUpperCase(),
-                            style: Theme.of(context).textTheme.subtitle1,
-                          ),
-                          const SizedBox(height: 5),
-                          Text(AppLocalizations.of(context).titleToday,
-                              style: Theme.of(context).textTheme.headline4),
-                        ],
-                      ),
-                      CircleAvatar(
-                        radius: 22,
-                        backgroundImage:
-                            NetworkImage('https://picsum.photos/250?image=9'),
-                      ),
-                    ],
-                  ),
+                  GetX<HomeController>(builder: (state) {
+                    return Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Column(
+                          mainAxisAlignment: MainAxisAlignment.start,
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text(
+                              state.currentDateText.value,
+                              style: Theme.of(context).textTheme.subtitle1,
+                            ),
+                            const SizedBox(height: 5),
+                            Text(AppLocalizations.of(context).titleToday,
+                                style: Theme.of(context).textTheme.headline4),
+                          ],
+                        ),
+                        CircleAvatar(
+                          radius: 22,
+                          backgroundColor: Colors.black12,
+                          foregroundImage:
+                              state.currentUserAvatar.value.isNotEmpty
+                                  ? NetworkImage(state.currentUserAvatar.value)
+                                  : AssetImage(
+                                      'assets/images/avatar_placeholder.png'),
```

Comment:
I believe that we will re-use the avatar placeholder in Drawer. How about creating constant variable for this string? 😄

Reply:
@sleepylee only one comment here 🙋

Reply:
updated, @manh-t 🙏